packages/file-storage/lib/index.ts

```diff
@@ -124,46 +104,51 @@ export class FileStorage {
     }
 
     const now = Date.now();
-    const issueAtDate = new Date(now - (now % (cacheSettings.signIssueTimeReductionModuloInSeconds * 1000)));
-    return this.client.presignedGetObject(this.fileStorageConfig.bucket, objectName, cacheSettings.expiryInSeconds, respHeaders, issueAtDate);
+    const requestDate = new Date(now - (now % (cacheSettings.signIssueTimeReductionModuloInSeconds * 1000)));
```

Comment:
Finde hier eigentlich das `issueAtDate` besser. Das was dabei rauskommt, ist nicht zwangsläufig das requestDate, sondern kann irgendein Zeitpunkt sein.

Reply:
Wird so in minio verwendet und auch beschrieben.

Reply:
Ich weis... Das konnte man früher gar nicht mitgeben. Das heißt bei minio noch so, weil sie es nicht ändern wollten. Richtig ist es trotzdem nicht.

Reply:
Done. `requestDate` -> `issueAtDate`